src/twisted/python/_setup.py

```diff
@@ -363,12 +363,6 @@ def _checkCPython(sys=sys, platform=platform):
 _isCPython = _checkCPython()
 
 notPortedModules = [
-    "twisted.internet.glib2reactor",
```

Comment:
This is uncovered in the tests because glib isn't available.  How do you know this is ported?  Or is the idea that removing these allows them to be ported.